fennel/datasets/datasets.py

```diff
@@ -111,7 +112,7 @@
 DEFAULT_INDEX_OFFLINE = "forever"
 DEFAULT_INDEX_ONLINE = True
 
-primitive_numeric_types = [int, float, pd.Int64Dtype, pd.Float64Dtype]
+primitive_numeric_types = [int, float, pd.Int64Dtype, pd.Float64Dtype, Decimal]
```

Comment:
all this will go with pandas 2.0 ?

Reply:
yup this will work with Pandas 2.0, basically this is for user provided dtype in the Dataset class.